server/test/unit/server/async/test_tasks.py

```diff
@@ -525,6 +527,7 @@ class EInfo(object):
             as one of the attributes. It stores string representation of traceback
             in it's traceback instance variable. This is a stub to imitate that behavior.
             """
+
```

Comment:
Yeah, it was IDE autoformatting.  Want me to remove them (and blacklist that rule)?